lib/http.js

```diff
@@ -232,6 +232,14 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
       const {profileAgent} = await profileAgents.get({id: profileAgentId});
 
       profileAgent.sequence++;
+      const now = Date.now();
+      const ttl = config['profile-http'].zcap.ttl;
+      for(const prop in zcaps) {
+        const zcap = zcaps[prop];
+        if(!zcap.expires) {
+          zcap.expires = new Date(now + ttl);
+        }
+      }
```

Comment:
I didn't realize this until just now, but this appears to just be adding `expires` to already created zcaps. That can't be done -- the `expires` field needs to be in the zcap when signed. I'm having trouble understanding why we need this PR at all -- if the zcaps are created with bedrock-profile.

Reply:
Well, we do need the validator changes, but this library shouldn't be trying to set default `expires` at all -- we can remove the above code and the config changes.